### PR TITLE
Release 1.4.1

### PR DESCRIPTION
**Pull request by jazzsequence**
This release adds:

* #270
* #271
* #273 

It addresses an issue that makes the WP CLI command added in 1.4.0 unusable in  WordPress multisite and clarifies the language in the notice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,21 @@ Added in 1.4.0. If you have run the `add-index` command and have verified that t
...
 
+### WordPress Multisite
+As of 1.4.1 the `add-index`, `primary-key-add` and `primary-key-revert` commands only apply to a single site. This means that to run on a WordPress multisite, for sites beyond the main site, you would need to pass the `--url=` flag for each subsite.
+
+However, you can script this process in bash by getting a list of sites and looping over them:
+
+```bash
+for site in $(wp site list --field=url);
+then
+	wp pantheon session add-index
```
Comment (rwagner00) [inline README.md:67]: Should this command use the site variable?

**Review by rwagner00 — APPROVED** [review]
Approved pending resolution of comment

File: README.md (hunk 1 above)
Comment (pwtyler) [reply README.md:67]:
```suggestion
	wp pantheon session add-index --url="$site"
```

Comment (jazzsequence) [reply README.md:67]: yerp it should. I'll need to open a new PR to make sure this gets merged to develop.

Comment (pwtyler) [reply README.md:67]: oh wait also it should be `for ... do ... done` not `then`

Comment (jazzsequence) [reply README.md]: fixed in #273

**Review by pwtyler — APPROVED** [review]

**Pull request merged**